lib/ui/pages/news_list/primary_news_list_item.dart

```diff
@@ -0,0 +1,116 @@
+import 'package:allthenews/ui/pages/news_list/primary_news_list_entity.dart';
+import 'package:flutter/material.dart';
+
+abstract class _Constants {
+  static const double newsImageRadius = 15;
```

Comment:
myślę czy jest sens jawnie wskazywać double , tj. czy jak dasz newsImageRadius = 15.0 to nie wywnioskuje, że to double?

Reply:
jak dam bez jawnego typu to mi sapie 
![image](https://user-images.githubusercontent.com/24854180/80869005-21eb1f80-8c9e-11ea-964c-c909b8aef114.png)

Reply:
racja, można pominąć